forge/apt.txt

```diff
@@ -0,0 +1 @@
+git
```

Comment:
maybe we should just install `git` from conda-forge in all the images...

Reply:
@scottyhq i'd actually recommend against that unless we have a specific reason, due to https://github.com/conda-forge/git-feedstock/issues/113 :(

Reply:
ah good to know!